content/open-source/index.rst

```diff
@@ -0,0 +1,20 @@
+public: yes
+location: open-source
+headline:
+  - tagline: 'Giving back to the community…'
+    type: 'Open-Source Contributions'
+after:
+  - include: 'projects/_feature.html.j2'
+    has: 'contributors'
+summary: |
+  **We use open-source software regularly**
+  in our personal and professional work,
+  to stay up-to-date with community best-practice
+  and avoid re-inventing any wheels.
+  We appreciate the generosity of so many in the open-source community,
+  and try to contribute back to the projects we use,
+  and share our own tools that might be useful to others.
+
+
+Open Tools for Developers
+=========================
```

Comment:
Should this list of OSS be combined with the `oss` list in `config.yml`? That is, it seems like these featured projects should also be included in the lists on the Birds page.